Userland/Services/WindowServer/WMConnectionFromClient.cpp

```diff
@@ -186,5 +186,4 @@ void WMConnectionFromClient::set_keymap(String const& keymap)
 {
     WindowManager::the().keymap_switcher()->set_keymap(keymap);
 }
-
```

Comment:
This is a (non style-conforming) change that gets reverted in a later commit, so that should not happen at all.